src/main/java/sirius/pasta/noodle/macros/PartMacro.java

```diff
@@ -27,7 +26,6 @@
  * this is a constant call and will be optimized away entirely.
  */
 @Register
```

Comment:
We currently use this at artefacts. Current usages could probably be changed/gitified. But for future quickfixes keeping this public would probably be helpful.

Reply:
done